src/app/Dao/IncomeDao.php

```diff
@@ -12,23 +12,25 @@ final class IncomeDao
 {
   /**
    * @var PDO
-   * @var string $incomeSourceId
    */
   private $pdo;
 
   /**
    * コンストラクタ
    * 
    * @param PDO $pdo
-   * @param 
    */
   public function __construct()
   {
     $this->pdo = new PDO("mysql:host=mysql; dbname=kakeibo; charset=utf8", "root", "password");
   }
 
   /**
-   * return array $user
+   * @param int $userId
+   * @param int $incomeSourced
+   * @param int $amount
+   * @param string $accrualDate
```

Comment:
このあたりもコメントを正確に整えちゃいましょう！
```suggestion
   * @param int $incomeSourceId
   * @param int $amount
   * @param string $accrualDate
```